### DISCUSSION TITLE
Tasmota Wi-Fi Range Extender - Hide access point SSID

### DISCUSSION THREAD

**Discussion by videsupra1** [arendst/Tasmota#17400] [Q&A] [answered]
Hi,

I have successfully set one of my plugs to work as Wi-Fi Range Extender according to this [article](https://tasmota.github.io/docs/Range-Extender) 
I want to achieve one more thing, but  I can not figure out how to hide SSID of AP that Tasmota present for devices to connect to. Is there any way to do this, is it actually possible?

**Comment by barbudor** [comment, marked as answer]
Tasmota do not support connecting to an hidden AP (and there are no plans to) 
So logically, there are no plans to allow an hidden SSID from the range extander

**Comment by v1-valux** [comment]
Technically, Tasmota is able to talk to hidden SSIDs, if you connect it to the same (visible) SSID before switching SSID to invisible.
I used to use my Tasmota devices like that.

It'd be good to have options I guess.
